packages/react-saasify/src/components/UndrawSVG/UndrawSVG.js

```diff
@@ -13,7 +13,7 @@ export class UndrawSVG extends Component {
     const { className, name, color, ...rest } = this.props
 
     const uri =
-      name.indexOf('/') >= 0 || name.indexOf('.') >= 0
+      name.includes('/') || name.includes('.')
```

Comment:
```js
const uri = ['/', '.'].some(char => name.includes(char)) ?
```

@transitive-bullshit which version seems more readable?

Reply:
let's keep as-is.